package.json

```diff
@@ -1,87 +1,88 @@
 {
-  "name": "at11",
```

Comment:
Here I think only the formatting was corrected... 4 spaces per tab instead of 2 spaces